packages/block-library/src/cover/shared.js

```diff
@@ -23,12 +25,34 @@ export function backgroundImageStyles( url ) {
 	return url ? { backgroundImage: `url(${ url })` } : {};
 }
 
+const isWeb = Platform.OS === 'web';
+
 export const CSS_UNITS = [
-	{ value: 'px', label: 'px', default: 430 },
-	{ value: 'em', label: 'em', default: 20 },
-	{ value: 'rem', label: 'rem', default: 20 },
-	{ value: 'vw', label: 'vw', default: 20 },
-	{ value: 'vh', label: 'vh', default: 50 },
+	{
```

Comment:
Did you remove the defaults intentionally?

Reply:
Totally no, good catch!